diff --git a/cmake/bflbsdk.cmake b/cmake/bflbsdk.cmake
index 8b7ceb2..65265aa 100644
--- a/cmake/bflbsdk.cmake
+++ b/cmake/bflbsdk.cmake
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.15)
 
+include(proj.conf OPTIONAL RESULT_VARIABLE proj_conf_file)
+
 if(CONFIG_BFLB_BOARD)
     set(BOARD ${CONFIG_BFLB_BOARD})
 endif()
@@ -10,6 +12,7 @@ if(CONFIG_BFLB_CPU_ID)
     set(CPU_ID ${CONFIG_BFLB_CPU_ID})
 endif()
 
+
 if(BOARD STREQUAL "ox64")
     set(CHIP "bl808")
 elseif(BOARD STREQUAL "m1sdock")
@@ -41,9 +44,6 @@ if(BOARD_DIR)
     message(STATUS "BOARD_DIR: ${BOARD_DIR}")
 endif()
 
-include(proj.conf OPTIONAL RESULT_VARIABLE proj_conf_file)
-
-
 find_package(bouffalo_sdk REQUIRED HINTS $ENV{BL_SDK_BASE})
 include(${SDK_PATH}/cmake/kconfig.cmake)
 
